### PR TITLE
a change to fix a bug with `/[\u{11450}\u{11C50}\u{11C52}]/u`

### PR DESCRIPTION
**Pull request by Yaffle**
see https://github.com/mathiasbynens/regexpu/issues/72

### CODE REVIEW

File: regenerate.js [hunk 1]
```diff
@@ -869,7 +869,8 @@
 				var otherLowSurrogateEnd = otherLowSurrogates[1];
 				if (
 					lowSurrogateStart == otherLowSurrogateStart &&
-					lowSurrogateEnd == otherLowSurrogateEnd
+					lowSurrogateEnd == otherLowSurrogateEnd &&
+					otherMapping[1].length === 2
```
Comment (Yaffle) [inline regenerate.js:873]: I am not sure about this code change, it is not obvious for me how the proper code change should look like, just random code code after surface looking at the code

Comment (mathiasbynens) [inline regenerate.js:873]:
```suggestion
					otherLowSurrogates.length === 2
```

**Comment by Yaffle** [comment]
@mathiasbynens , thanks, now it works for `/\p{Decimal_Number}/u`

**Comment by mathiasbynens** [comment]
Thanks for the fix!

**Pull request merged**